java/core/src/main/java/sleeper/core/properties/table/TablePropertiesStore.java

```diff
@@ -60,8 +61,9 @@ TableProperties loadProperties(TableStatus table) {
     /**
      * Loads and validates properties for the Sleeper table with the given name.
      *
-     * @param  tableName the table name
-     * @return           the table properties
+     * @param  tableName              the table name
+     * @return                        the table properties
+     * @throws TableNotFoundException if the table for the given name is not found.
```

Comment:
This description doesn't need a full stop, as it's not a sentence.

Should this be "with the given name" rather than "for the given name"?